compiler/rustc_trait_selection/src/traits/const_evaluatable.rs

```diff
@@ -138,10 +138,10 @@ pub fn is_const_evaluatable<'tcx>(
                 } else if uv.has_non_region_param() {
                     NotConstEvaluatable::MentionsParam
                 } else {
-                    let guar = infcx.tcx.sess.delay_span_bug(
-                        span,
-                        format!("Missing value for constant, but no error reported?"),
-                    );
+                    let guar = infcx
```

Comment:
Interesting how `rustfmt` changed its heuristics here, I guess the long macro was keeping it from doing this (arguably the old formatting was better imo but whatever, can't change that).